extensions/amp-story-dev-tools/0.1/amp-story-dev-tools-tab-preview.css

```diff
@@ -0,0 +1,243 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+.i-amphtml-story-dev-tools-device {
+  background: #101010 !important;
+  border-radius: 40px !important;
+  box-shadow: 0px 11.0046px 27.5114px rgba(0, 0, 0, 0.34), inset 0px 2.20091px 3.30137px rgba(44, 46, 52, 0.6) !important;
+  display: flex !important;
+  flex-direction: column !important;
+  overflow: hidden !important;
+  position: absolute !important;
+  margin: auto !important;
+  height: fit-content !important;
+  top: 0 !important;
+  bottom: 0 !important;
+  padding: 10px 8px !important;
+  height: fit-content !important;
+  opacity: 0;
+  transition: opacity 0.15s ease-in-out;
+}
+
+.i-amphtml-story-dev-tools-device-screen {
+  border-radius: 30px !important;
+  background: #2E3137 !important;
+  overflow: hidden !important;
+  padding-top: 40px;
```

Comment:
Why no !important? Should this be a `margin`?

Reply:
Forgot the important, will add it. This is a temporary space between the top of the device and the player, where the app UI would be, so it's a padding on the screen (or could be a margin on the player). When we add the device UI that Hong designed, the padding will be replaced with actual UI.